nxdrive/client/local/linux.py

```diff
@@ -24,8 +26,19 @@ class LocalClient(LocalClientMixin):
 
     def has_folder_icon(self, ref: Path) -> bool:
         """Check if the folder icon is set."""
-        # To be implementation with https://jira.nuxeo.com/browse/NXDRIVE-1831
-        return True
+        matcher = re.compile(r"metadata::emblems: \[nuxeo.*\]")
+        folder_path = self.abspath(ref).as_posix()
```

Comment:
On GNU/Linux, `.as_posix()` is not needed. You could then move that variable directly into the `subprocess` call.